JAICore/jaicore-basic/src/test/java/jaicore/testproblems/npuzzle/NPuzzleState.java

```diff
@@ -61,63 +61,57 @@ public NPuzzleNode(int dim, int seed) {
 	 * 			The empty space on the x-axis.
 	 * @param emptyY
 	 * 			The empty space on the y-axis.
-	 * 
+	 *
 	 * @param noMoves
 	 * 			The number of already done moves.
 	 */
-	public NPuzzleNode(int [][] board, int emptyX, int emptyY, int numberOfMoves) {
+	public NPuzzleState(final int [][] board, final int emptyX, final int emptyY) {
 		this.board = board;
 		this.emptyX = emptyX;
 		this.emptyY = emptyY;
-		this.numberOfMoves = numberOfMoves;
 	}
 
-	
+
 	public int[][] getBoard() {
-		return board;
+		return this.board;
 	}
 
 	public int getEmptyX() {
-		return emptyX;
+		return this.emptyX;
 	}
 
 	public int getEmptyY() {
-		return emptyY;
-	}
-	
-	public int getNumberOfMoves() {
-		return this.numberOfMoves;
+		return this.emptyY;
 	}
-	
-	
+
 	/**
-	 * Returns a graphical version of the board configuration. 
+	 * Returns a graphical version of the board configuration.
 	 * Works best if there is no number with two or more digits.
 	 */
 	@Override
 	public String toString() {
-//		return "NPuzzleNode [board=" + Arrays.toString(board) + ", emptyX=" + emptyX + ", emptyY=" + emptyY + "]";
+		//		return "NPuzzleNode [board=" + Arrays.toString(board) + ", emptyX=" + emptyX + ", emptyY=" + emptyY + "]";
```

Comment:
Code in the comments: If this representations might be used in the future, maybe we could introduce a flag to toggle the different kinds of outputs for the toString?

Reply:
good point, I now just removed this part. If one wants to come back to this, a toggle variable would be a good solution.